snuba/migrations/snuba_migrations/metrics/0001_metrics_buckets.py

```diff
@@ -0,0 +1,75 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    Array,
+    Column,
+    DateTime,
+    Float,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: Sequence[Column[Modifiers]] = [
+    Column("org_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("metric_id", UInt(64)),
+    Column("metric_type", String(Modifiers(low_cardinality=True))),
+    Column("timestamp", DateTime()),
+    Column("tags", Nested([Column("key", UInt(64)), Column("value", UInt(64))])),
+    Column("sum_values", Float(64)),
+    Column("distribution_values", Array(Float(64))),
+    Column("set_values", Array(UInt(64))),
+    Column("materialization_version", UInt(8)),
```

Comment:
Let's see if we can do 0 downtime migrations of the materialized view. 

the matview statement will have a condition based on this field so we could have two or more matviews existing at the same as long as the conditions are mutually exclusive. 
That allows us to switch from one to the other without stopping the consumer